metricbeat/module/system/diskio/diskstat_linux_test.go

```diff
@@ -0,0 +1,14 @@
+// +build linux
+
+package diskio
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_Get_CLK_TCK(t *testing.T) {
+	//usually the tick is 100
+	assert.Equal(t, uint32(100), Get_CLK_TCK())
```

Comment:
er, in theory, the CLK_TCK should be got using `sysconf(C._SC_CLK_TCK)`, need cgo enabled. and this should included in cpu module. but in order to calculate disk metrics, i have to know the cpu clock ticks, and hard code the value to `100`.